lib/Sema/MiscDiagnostics.cpp

```diff
@@ -59,8 +59,8 @@ using namespace constraints;
 /// Return true if this expression is an implicit promotion from T to T?.
```

Comment:
FYI @AnirudhMathur12, regarding the unintended reformatting here – there is a [section in the FAQ](https://github.com/swiftlang/swift/blob/652402e9e83d2e970de5a12c34e419f95d1cb7f3/docs/HowToGuides/FAQ.md#how-do-i-format-my-changes) regarding how to use `git-clang-format` to only reformat the altered portion of files in a git changeset. that tool works well for the c++ code in this codebase.